app/components/registryBrowseTags/registryBrowseTagsController.js

```diff
@@ -0,0 +1,93 @@
+angular.module('registryBrowseTags', [])
+.controller('RegistryBrowseTagsController', ['$q', '$scope', '$transition$', 'RegistryService', 'Notifications',
+function ($q, $scope, $transition$, RegistryService, Notifications) {
+
+  $scope.state = {};
+  $scope.tags = [];
+
+  var blobs_size = {};
+
+  function initView() {
+    var registryID = $transition$.params().id;
+    var repository = $transition$.params().repository;
+    $scope.repository = repository;
+
+    RegistryService.tags(registryID, repository)
```

Comment:
The content of this method is pretty complex. We'll need to find a way to refactor it and maybe abstract it inside a service. At least it should be divided in multiple smaller methods.

Reply:
Aggree